practice-one/src/components/Button/Button.jsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import "./Button.css";
+import PropTypes from "prop-types";
+
+const Button = (props) => {
+  const {
+    type,
+    children,
+    width,
```

Comment:
why need to pass width, height, color, fontSize, fontWeight, border and align?

Reply:
Because there are many different buttons in the design, splitting it up like this makes it easier for me to control its properties.

Reply:
need to create a type button, just pass type, it'll include all style follow the type